hyperspy/_signals/lazy.py

```diff
@@ -278,6 +325,11 @@ def _get_navigation_chunk_size(self):
         nav_chunks = tuple([self.data.chunks[i] for i in sorted(nav_axes)])
         return nav_chunks
 
+    def _get_signal_chunk_size(self):
```

Comment:
Maybe it would be worth to refactor `_get_signal_chunk_size` and `_get_navigation_chunk_size` to make it more generic and a public API to address #1931 at the same time?

For example, something along the line of:
```python
# the get the chunk size of the signal axes
s.get_chunk_size(s.axes_manager.signals_axes)

# the get the chunk size of the navigation axes
s.get_chunk_size(s.axes_manager.navigation_axes)

# to get the chunk size of any axis, for example axis 2 in the axes_manager
s.get_chunk_size(s.axes_manager[2])
```